### PR TITLE
BLOG added in the navbar web3community#33

### PR DESCRIPTION
**Pull request by 0shivamh**
## Related Issue 

- Information about the related issue 

Closes: #33 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,11 +47,14 @@
         <div class="w-full flex-grow lg:flex lg:items-center lg:w-auto hidden mt-2 lg:mt-0 bg-white lg:bg-transparent text-black p-4 lg:p-0 z-20" id="nav-content">
           <ul class="list-reset lg:flex justify-end flex-1 items-center">
             <li class="mr-3">
-              <a class="inline-block py-2 px-4 text-blue-800 font-bold no-underline" href="#">HOME</a>
+              <a class="inline-block py-2 px-4 text-blue-800 font-bold no-underline" href="/">HOME</a>
```
Comment (vinzvinci) [inline index.html:50]: Why change to slash? Can u replace it with the hash again?

Comment (0shivamh) [reply index.html:50]: okay, wait

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by Panquesito7 — APPROVED** [review]
For me, this is looking good. Thank you! 😄👍

**Pull request merged**